src/Runtime.Base/src/System/Runtime/ExceptionHandling.cs

```diff
@@ -410,19 +398,20 @@ internal enum ExKind : byte
             None = 0,
             Throw = 1,
             HardwareFault = 2,
-            // unused: 3
             KindMask = 3,
 
             RethrowFlag = 4,
-            Rethrow = 5,        // RethrowFlag | Throw
-            RethrowFault = 6,   // RethrowFlag | HardwareFault
+
+            SuperscededFlag = 8,
```

Comment:
What is this SuperscededFlag good for? It doesn't seem to be used.
Also, the spelling should be SupersededFlag, right?